shub/deploy.py

```diff
@@ -1,25 +1,26 @@
 from __future__ import absolute_import
-import os
+
 import glob
+import json
+import os
 import shutil
 import tempfile
-import json
-from six.moves.urllib.parse import urljoin
+from typing import AnyStr, Optional, Union
 
 import click
-import toml
 # Not used in code but needed in runtime, don't remove!
 import setuptools
 import setuptools.msvc  # noqa
+import toml
```

Comment:
I appreciate the attempt to clean up the imports, however let's keep the whiteline after the setuptools imports - the comment above only applies to them.

Reply:
Sorry, it is an addiction that I have to format the code after making any changes. I have already returned it to the previous state.